falcon_multiqc/commands/query.py

```diff
@@ -296,8 +303,16 @@ def cli(
 
     # Sqlaclehmy query that will be constructed based on this command's options.
     falcon_query = None
+    
+    # Check output and filename for validity.
     if (output):
         output = os.path.abspath(output)
+        if (not os.path.exists(os.path.dirname(output))):
+            raise Exception(f"Output path {os.path.dirname(output)} does not exist.")
+        if (not os.path.isdir(output)):
+            raise Exception(f"Output path {os.path.dirname(output)} is NOT a directory. Please use a directory path with --output.")
+        if not filename:
+            raise Exception("--output requires --filename to name the csv or multiqc report")
```

Comment:
```suggestion
    if (output):
        output = os.path.abspath(output)
        if (not os.path.isdir(output)):
            raise Exception(f"Output path {output} is NOT a directory. Please use a directory path with --output.")
        if (not os.path.exists(output)):
            raise Exception(f"Output path {os.path.dirname(output)} does not exist.")        
        if not filename:
            raise Exception("--output requires --filename (no extension) to name the csv or multiqc report")
```

Hey I think this layout presents the exceptions a little more accurately in stdout, but apart from that It's great